Backpack-SwiftUI/InsetBanner/Classes/BPKInsetBanner.swift

```diff
@@ -42,19 +42,19 @@ public struct BPKSponsoredBanner: View {
         isExpanded: Bool = false,
```

Comment:
This probably should be a binding if we want it to be outside controlled.

Reply:
Good point! I think it's mostly only for initialising it from a certain state for testability purposes. But happy to change to `@Binding` .